README.md

```diff
@@ -28,8 +28,12 @@ gulp.task('css', function() {
 #### ruleConfiguration
 Type: `Object`
 
+If you pass `lookup: false`, the local .csslintrc is not looked up automatically.
```

Comment:
They don't currently have one, so I'm not too worried: https://github.com/CSSLint/csslint/wiki/Rules-by-ID

However, we could prefix it with an underscore or accept `false` as a second argument to avoid lookup... Thoughts?